solution.js

```diff
@@ -38,8 +171,19 @@ const catArtSortByPriceA = () => {};
 // or try to implement merge sort
 // or look up another common sort algorithm (i.e. quicksort, ) and try your own implementation
 // Bonus add another argument that would allow the function to be used for ascending or descending order
-const mySortFunction = () => {};
-
+const mySortFunction = (data) => {
+  let mySorting  = true
+while (mySorting){
+    mySorting = false;
+    for (let i = 0; i < data.length-1; i++){
+        if (data[i] > data[i+1]){
+            [data[i], data[i+1]] = [data[i+1], data[i]]
```

Comment:
Bubble sort is a great sort to remember because of its ease to implement but its not great  in terms of time complexity.  

Try to implement a merge or. a quick